go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cert-manager/approver-policy
 
-go 1.23.0
+go 1.24.0
```

Comment:
/hold Won't dependabot still add that toolchain line in #622, unless we set this to 1.24.2?

Reply:
No, I think not. But there is only one way to find out. 😉 I have reported toolchain issues to Dependabot multiple times, but the bugs keep coming back.

Reply:
The latest weeks, Dependabot seems to only add toolchain if it needs to bump the Go version. And I don't think 1.24.2 is required by our (updated) dependencies. It's just the version we use.

Reply:
Can I unhold to see what happens?